concent-builder/role-tiller.yml

```diff
@@ -0,0 +1,49 @@
+# Tiller permission to all resources in default namespace
```

Comment:
Please reword and put the command near the binding:

`Give tiller complete control over the default namespace`

Reply:
Done.